### PR TITLE
fix(public-cloud.privatenetwork): change guide links

### PR DESCRIPTION
**Pull request by stif59100**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/sprint-2250-2252`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-10342
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I change guide link for private network.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ export const VLAN_ID = {
 
 export const GUIDE_LINKS = {
   PRIVATE_NETWORK_WITH_GATEWAY:
-    'https://docs.ovh.com/gb/en/public-cloud/creating-private-network-with-gateway/',
+    'https://docs.ovh.com/fr/publiccloud/network-services/creating-private-network-with-gateway/',
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:16]: **question:** Are we sure to include only French pages? If i am not wrong, these docs are available in other languages as well.

Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:16]: I think as usual we need based on subsidiary.

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:16]: as highlight by Guillaume, better to have a subs list.

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js [hunk 2]
```diff
@@ -32,6 +32,13 @@ export default class PrivateNetworksAddCtrl {
     this.GUIDE_LINKS = GUIDE_LINKS;
     this.VLAN_ID = VLAN_ID;
     this.TRACKING_PREFIX = TRACKING_PREFIX;
+    this.UrlGuidePrivate =
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js:35]:
```suggestion
    this.privateNetworkUrl =
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 3]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+    US: `${helpDocRoot}us/en${urlPrivate}`,
+  },
+  REGION_AVAILABILITY: {
+    DEFAULT: `${helpRoot}gb/en${urlRegion}`,
+    DE: `${helpRoot}de${urlRegion}`,
+    ES: `${helpRoot}es${urlRegion}`,
+    IT: `${helpRoot}it${urlRegion}`,
+    PL: `${helpRoot}pl${urlRegion}`,
+    PT: `${helpRoot}pt${urlRegion}`,
+    QC: `${helpRoot}fr${urlRegion}`,
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:48]:
```suggestion
    QC: `${helpRoot}fr-ca${urlRegion}`,
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js [hunk 4]
```diff
@@ -32,6 +32,13 @@ export default class PrivateNetworksAddCtrl {
...
+      GUIDE_LINKS.PRIVATE_NETWORK_WITH_GATEWAY[
+        coreConfig.getUser().ovhSubsidiary
+      ] || GUIDE_LINKS.PRIVATE_NETWORK_WITH_GATEWAY.DEFAULT;
+    this.UrlGuideRegion =
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js:39]:
```suggestion
    this.regionsGuideUrl =
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 5]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+    MA: `${helpRoot}fr${urlRegion}`,
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:49]:
```suggestion
    MA: `${helpRoot}fr-ma${urlRegion}`,
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 6]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+    SN: `${helpRoot}fr${urlRegion}`,
+    TN: `${helpRoot}fr${urlRegion}`,
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:51]:
```suggestion
    TN: `${helpRoot}fr-tn${urlRegion}`,
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 7]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
   MAX: 4000,
 };
 
+const helpDocRoot = 'https://docs.ovh.com/';
+const helpRoot = 'https://www.ovhcloud.com/';
+const urlPrivate =
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:16]:
```suggestion
const privateNetwrokUrl=
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 8]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+  REGION_AVAILABILITY: {
+    DEFAULT: `${helpRoot}gb/en${urlRegion}`,
+    DE: `${helpRoot}de${urlRegion}`,
+    ES: `${helpRoot}es${urlRegion}`,
+    IT: `${helpRoot}it${urlRegion}`,
+    PL: `${helpRoot}pl${urlRegion}`,
+    PT: `${helpRoot}pt${urlRegion}`,
+    QC: `${helpRoot}fr${urlRegion}`,
+    MA: `${helpRoot}fr${urlRegion}`,
+    SN: `${helpRoot}fr${urlRegion}`,
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:50]:
```suggestion
    SN: `${helpRoot}fr-sn${urlRegion}`,
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 9]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
   MAX: 4000,
 };
 
+const helpDocRoot = 'https://docs.ovh.com/';
+const helpRoot = 'https://www.ovhcloud.com/';
+const urlPrivate =
+  '/publiccloud/network-services/creating-private-network-with-gateway/';
+const urlRegion = '/public-cloud/regions-availability/';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:18]:
```suggestion
const REGION_AVAILABILITY_URL = '/public-cloud/regions-availability/';
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 10]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
   MAX: 4000,
 };
 
+const helpDocRoot = 'https://docs.ovh.com/';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:14]:
```suggestion
const HELP_DOC_ROOT = 'https://docs.ovh.com/';
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 11]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+const helpRoot = 'https://www.ovhcloud.com/';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:15]:
```suggestion
const HELP_ROOT = 'https://www.ovhcloud.com/';
```

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js [hunk 12]
```diff
@@ -11,11 +11,46 @@ export const VLAN_ID = {
...
+    QC: `${helpDocRoot}ca/fr${urlPrivate}`,
+    MA: `${helpDocRoot}fr${urlPrivate}`,
+    SN: `${helpDocRoot}fr${urlPrivate}`,
+    TN: `${helpDocRoot}fr${urlPrivate}`,
+    AU: `${helpDocRoot}au/en${urlPrivate}`,
+    SG: `${helpDocRoot}sg/en${urlPrivate}`,
+    FR: `${helpDocRoot}fr${urlPrivate}`,
+    US: `${helpDocRoot}us/en${urlPrivate}`,
+  },
+  REGION_AVAILABILITY: {
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js:41]: Some subs are missed. There are more.

File: packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js [hunk 13]
```diff
@@ -32,6 +32,13 @@ export default class PrivateNetworksAddCtrl {
...
+      GUIDE_LINKS.REGION_AVAILABILITY[coreConfig.getUser().ovhSubsidiary] ||
+      GUIDE_LINKS.REGION_AVAILABILITY.DEFAULT;
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js:41]:
```suggestion
    this.subsidiary = coreConfig.getUser().ovhSubsidiary;
    this.privateGuideUrl =
      GUIDE_LINKS.PRIVATE_NETWORK_WITH_GATEWAY[
        this.subsidiary
      ] || GUIDE_LINKS.PRIVATE_NETWORK_WITH_GATEWAY.DEFAULT;
    this.regionGuideUrl =
      GUIDE_LINKS.REGION_AVAILABILITY[this.subsidiary] ||
      GUIDE_LINKS.REGION_AVAILABILITY.DEFAULT;
```

**Review by ghyenne — CHANGES_REQUESTED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**